UPGRADE.md

```diff
@@ -1,5 +1,9 @@
 # Upgrade to 2.9
 
+## MINOR BC BREAK `Statement::execute()`.
+
+Similarly to the drivers based on `pdo_pgsql` and `pdo_sqlsrv`, `OCI8Statement::execute()` and `MySQLiStatement::execute()` do not longer ignore redundant parameters.
```

Comment:
This is a major BC break, since apps built on these will really blow up spectacularly

Reply:
I marked it as minor because it will only occur in the case of a logic error in application code. This was exactly my case. The same code already doesn't work with other drivers. Do you suggest to change the wording or move it to `develop`?

In a similar issue https://github.com/doctrine/dbal/pull/3359, @deeky666 [suggested](https://github.com/doctrine/dbal/pull/3359#discussion_r237781659) to implement exceptions and document it as a minor BC break.

I'm confused.

Reply:
Yeah, while it is indeed an application error, consumers may have built code that just stuffs literally everything in query parameters, so we are actively breaking their stuff.

I think this should be moved to major 😣